src/mod_auth_gssapi.c

```
@@ -1706,6 +1706,23 @@ static const char *mag_basic_auth_mechs(cmd_parms *parms, void *mconfig,
 }
 #endif
 
+static const char *mag_acceptor_name(cmd_parms *parms, void *mconfig,
+                                     const char *w)
+{
+    struct mag_config *cfg = (struct mag_config *)mconfig;
+    gss_buffer_desc bufnam = { strlen(w), (void *)w };
+    uint32_t maj, min;
+
+    maj = gss_import_name(&min, &bufnam, GSS_C_NT_HOSTBASED_SERVICE,
+                          &cfg->acceptor_name);
+    if (GSS_ERROR(maj)) {
+        ap_log_error(APLOG_MARK, APLOG_WARNING, 0, parms->server,
+                     "gss_import_name([%s]) failed", w);
```

Return non-null, so the directive won't get silently ignored.